whirlpool/appliance.py

```diff
@@ -139,6 +146,9 @@ def get_attribute(self, attribute):
         return self._data_dict["attributes"][attribute]["value"]
 
     def has_attribute(self, attribute):
+        if self._data_dict is None:
+            LOGGER.error("No data available")
+            return None
```

Comment:
```suggestion
            return False
```
lets keep this func boolean